### DISCUSSION TITLE
Discussing experimental  (virtual?) communities / interest groups

### DISCUSSION THREAD

**Discussion by martinprogrammer** [life-itself/community#909] [General]
**Update**: Focal topic is ...

> Creation of an Expats community that's aimed at members being able to choose a geographical location and set up a DAO in order to jointly research, select & purchase property to develop (or renovate) into residential units. They'll be able to make significant savings by centrally employing developers, architects and the legal team and own a house/apartment/room within the same complex (having like-minded neighbours/friends around them).

---

Let's start a conversation - any of the questions below or anything related :-)

What are the ingredients needed to build progressive communities defined by common goals? 
What are strategies for defining common goals and subscribing to them? 
What are the practical operational aspects of running such communities?
What are the conflict resolution options?
What are the limits of members' collaboration?
How to nurture real friendship and kinship in such communities?

How to homogenise the members and how important is that?

What would an ideal community look like if taken from a future concept and worked backwards towards our reality?
What would an ideal community look like if we imagined it evolved forward from here?

What are the ways of maintaining high levels of motivation?
How would a community leverage neuro-divergent people?
Would members benefit by making financial decisions together or financing projects jointly?
How would communities exert political power?

How would communities be financed?

**Comment by rufuspollock** [comment]
@martinprogrammer great questions. It would help folks in to have a specific example community or project in mind. it could even be  a toy or made-up example. Could you provide one?

**Reply by martinprogrammer** [reply]
@rufuspollock - I'm looking for a blueprint that can be used to spin off quick communities/interest groups in order to learn about them.  More like experimental groups, rather than an attempt at individual longevity.  Then, I'd like to know if it's possible to combine these groups dynamically.  How to attract quality members and whether this "quality" is generic - i.e. transferrable to other communities.  Also, whether the collectiveness of a community can spawn a brand new "shared purpose" that can evolve selectively into another community.  In other words, is there an evolutive process that could be distilling the cohort as it transitions into a "new" community with the remnants of the old community.  I'd like to know what aspects of these "mushroom" communities can be done at scale (be automated) and what needs to be intentional creative work.  Then there's motivational side to it - is it the friendship and people liking each other more important than the shared purpose in terms of community "glue".  How to ensure members' participation?  Psychological aspects of engagement and how to tackle them.  Why in all these groups only (let's say) 5% of the members participate and the rest quietly observing from afar?  I'd like to kick start conversation to see if there's any interest in this before following it up with specific examples.

**Reply by rufuspollock** [reply]
@martinprogrammer i think i understand. Can you still give a concrete example of an actual community or interest group you'd like to "spin off"?

**Reply by martinprogrammer** [reply]
@rufuspollock  It could be anything - 
1) a community of film makers creating content in the space of activism of some sort
2) a community that gets together in order to socialise and play D&D
3) a group of people putting money wanting to invest together
4) a self help community, a mentoring community...
5) a community dedicated to futurism and social modelling
6) an Expats community...

Grab any issue that motivates people.  Combine that with your research and have controlled cohorts adhering to certain "generic" rules to nurture a certain type of enviornment...

**Reply by rufuspollock** [reply]
@martinprogrammer great. Let's pick one that is most relevant to you ... which one is it?

**Reply by martinprogrammer** [reply]
What is it that we are doing here?  
Discussing or actually spinning off a community?  Do we want to make money, fight injustice or do a bit of world building?  

Maybe build on your co-habiting theme?

We could do an Expats community that's aimed at members being able to choose a geographical location and set up a DAO in order to jointly research, select & purchase property to develop  (or renovate) into residential units.  They'll be able to make significant savings by centrally employing developers, architects and the legal team and own a house/apartment/room within the same complex (having like-minded neighbours/friends around them).  

Various projects would be able to collaborate in terms of "collective knowledge" and move onto new projects in different setups.

Would that be a good starting point?

**Reply by rufuspollock** [reply]
> We could do an Expats community that's aimed at members being able to choose a geographical location and set up a DAO in order to jointly research, select & purchase property to develop (or renovate) into residential units. They'll be able to make significant savings by centrally employing developers, architects and the legal team and own a house/apartment/room within the same complex (having like-minded neighbours/friends around them).

So i'm clear: is this your chosen example you want to focus on? 😄👍

**Reply by martinprogrammer** [reply]
Sure. if you don't like it, we could dissect another one.

**Reply by rufuspollock** [reply]
Great. Let's go with that ... I've updated the main description. Now let's pick a first question ...

**Comment by rufuspollock** [comment]
## Question 1: What are the ingredients needed to build this community defined by common goals?

First, to recap, the community is ...

> An Expats community that's aimed at members being able to choose a geographical location and then purchase and/or build a set of residential units ...

So this is a classic "let's start a (conscious) coliving community" project ...

So what are you going to need?

- People, at least a core group of 2-6 people to start the project
- Capital (unless you go a pure lease model)
- Patience - this project will take 2-5 years

In the core group you are going to want a significant alignment on vision and values.

I'd recommend reading this excellent book for more: https://lifeitself.org/blog/2021/05/14/notes-on-creating-a-life-together-by-diana-leafe-christian

**Reply by rufuspollock** [reply]
@martinprogrammer let's discuss this response and then we can do more questions ...

**Reply by martinprogrammer** [reply]
Maybe we need to refine the mission statement?  Also, should we envisage that the community's common purpose / goals might pivot once the the ball gets rolling?  In that case, we could prepare ourselves by modelling different scenarios.  

Regarding the funding - Apart from the funding of the project in terms of setting up the community, is there any other upfront funding?  I suppose that Incorporation of DAOs can take place later.   Please correct me if I'm wrong - the community is not inside a DAO, but only projects.

I'll have a look at the book and continue the convo later today :-)

**Reply by rufuspollock** [reply]
@martinprogrammer yes, let's refine the mission statement. And, of course, that mission statement can evolve once the community is running.

> Regarding the funding - Apart from the funding of the project in terms of setting up the community, is there any other upfront funding? I suppose that Incorporation of DAOs can take place later. Please correct me if I'm wrong - the community is not inside a DAO, but only projects.

I'm not sure funding will be that important for the very first step of getting a group of interested people together.

On the DAO point: i think that is a side issue. IMO whenever you read DAO you can just reading "corporate entity" be that an association, a LLC/LTD etc.

**Reply by martinprogrammer** [reply]
@rufuspollock  - I'm tossing around some ideas, let me know if it makes sense or needs a re-think

Shall we say that the mission is a "curated" community of expats who will be able to co-operate in various stages of their project (which is defined as a move to a particular location in order to start a new life or establish an alternative living location).  

The stages:
1) Getting in touch with people who share the same objectives in order to exchange the ideas, enthusiasm and knowledge
2) Researching various aspects of life abroad (immigration, work, culture, education, language)
3) Researching properties and buying process
4) Joining together to buy properties
5) Joining together to develop properties
6) Joining together to simplify various aspects of bureaucracy and obtaining services , getting discount on insurance, creating a pool of service providers, car share, various exchange of favours within the group
7) Socialising

I think it is important for us to brainstorm all these potential needs and package them into features.
Those "novel" ways of interacting will still need to be "sold" to the members as viable alternative solutions to their problems.
The segment of the community who would be prepared to get into format agreements of points 4 and 5 will probably be very small to start with.

For the "getting a group of interested people bit" - is it a matter of creating a portal and then marketing it in a conventional manner?

Regarding  the DAO - I agree, it's just a technology that stands for an organisation or a company.

**Reply by rufuspollock** [reply]
I like the iteration:

> "curated" community of expats who will be able to co-operate in various stages of their project (which is defined as a move to a particular location in order to start a new life or establish an alternative living location).

I think the addition of *curated* is useful and also like that this is a group of people who each may have their own projects but want to cooperate and learn from each other.

I think the steps also make a lot of sense. Even more simply you have something:

1. Connection and peer learning
2. Collaborating on initiatives e.g. some people in community starting an effort together

I think this has a lot of viability and is sort of what we have and are doing with the conscious coliving groups and DDS efforts.

**Reply by rufuspollock** [reply]
@martinprogrammer we can talk about what are the first next steps here if you want ...

**Reply by martinprogrammer** [reply]
@rufuspollock 

Sure, one thing that bugs me at the moment is whether integrating a commercial model into it would compromise the project's fundamental ethos.  

What do you think?

It would be nice to strike a balance between community-driven initiatives and introducing elements that could sustain the project financially. How can we effectively merge these aspects to ensure the project remains true to its core values while exploring potential commercial opportunities?  

Does this come across as a reasonable question?

**Reply by rufuspollock** [reply]
@martinprogrammer you definitely want to think of a sustainability model. that said i think i'd do a non-commercial pilot to see if the basic idea even works and what t he value proposition is.

**Comment by martinprogrammer** [comment]
@rufuspollock 
I've read the blogpost/booklet.  Some really good points around minimising ideals/values that can possibly clash with each other and become self-destructing seeds. 

The understanding of psychological underpinnings characterised by a different mental wiring and manifested in diverse social requirements, values prioritisation, conflict resolution styles, information processing and work approaches is one of those essential components.

Another one is not to fall into a trap of idealism and ignore the realities and the research into the reasons of communities' failures.
The rules of a community should be set out clearly and "policed" in a non-discretionary manner - as to draw boundary lines.

This one particularly resonates with me:  One bad apple can destroy a community (and even if doesn't destroy it will eat a lot of energy).  I believe that communities need to be carefully put together (a bit like a team building exercise) and curated.  Education plays a great role here - a community needs to be homogenised through specially designed classes instilling common values and the purpose and inviting challenging views (if nothing else to highlight as soon as possible that the ideals are not shared).

**Reply by rufuspollock** [reply]
Yes, good points. And the book itself is definitely worth skimming - it is in our library https://lifeitself.org/tao/library.

> This one particularly resonates with me: One bad apple can destroy a community (and even if doesn't destroy it will eat a lot of energy). I believe that communities need to be carefully put together (a bit like a team building exercise) and curated. Education plays a great role here - a community needs to be homogenised through specially designed classes instilling common values and the purpose and inviting challenging views (if nothing else to highlight as soon as possible that the ideals are not shared).

Yes, and that's not that easy to do especially early on when there is a tension between the desire to find new participants and the need to be rigorous in admittance.
